docs/components/table.md

```diff
@@ -1270,12 +1270,10 @@ Set the following properties to handle empty state:
 
 - `emptyStateTitle` - Title text for empty state
 - `emptyStateMessage` - Message for empty state
-- `emptyStateIcon` - Icon for empty state
-- `emptyStateIconColor` - Color for empty state icon
-- `emptyStateIconSize` - Size for empty state icon
 - `emptyStateActionRoute` - Route for empty state action
 - `emptyStateActionMessage` - Button text for empty state action
-- `emptyStateActionButtonIcon` - Icon for the empty state action button
+- `emptyStateActionButtonShowPlusIcon` - Boolean prop to show/hide plus icon in empty state action button
```

Comment:
Is this really a prop? 34 character prop name...

Reply:
name it something more concise

Reply:
You could try something to indicate the usage instead of what it is like: `emptyStateCreationActionButton` - this is used on create CTAs

Reply:
How about `emptyStateActionCreation`?

Reply:
`actionButtonPlusIcon: boolean` ?

Reply:
What if we just simplified for KTable and didn't show an icon in the empty state? Wouldn't that make things way easier? Surely a user is smart enough to read the button and see that it says `Add {entity}`? 🤷🏼 

Again, this seems extremely use-case specific for Konnect.

Reply:
I feel like it's not our decision to make.. maybe slot will be a better alternative? E.g. `action-button-text` that would take precedence over `actionButtonText` prop if provided.

Reply:
action button slot - then you have to slot the whole button? 
I don't think adding a prop for a component nested 3 levels deep is ideal

Reply:
Okay, scratch that - removed icon-related props and slots - we're just gonna use `empty-state` slot to slot in the whole `KEmptyState` component with all needed buttons and icons